### PR TITLE
feat: Add push/email notification support for SLA

### PR DESCRIPTION
**Pull request by vishnu-narayanan**
# Pull Request Template

## Description

image

## Changelog

- Add push/email settings for SLA related notifications to profile page
- Hide the settings if feature not enabled

Fixes https://linear.app/chatwoot/issue/CW-3152/push-email-notification-support-for-sla

## Type of change

Please delete options that are not relevant.

- [x] New feature (non-breaking change which adds functionality)

## How Has This Been Tested?

- Tested locally

image

image

## Checklist:

- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my code
- [ ] I have commented on my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

**Review by sojan-official — APPROVED** [review]
Code LGTM

File: app/views/mailers/agent_notifications/conversation_notifications_mailer/sla_missed_first_response.liquid [hunk 1]
```diff
@@ -0,0 +1,11 @@
+<p>Hi {{user.available_name}},</p>
+
+{{ byebug }}
```
Comment (muhsin-k) [inline app/views/mailers/agent_notifications/conversation_notifications_mailer/sla_missed_first_response.liquid:3]: Intentional?

File: app/drops/sla_policy_drop.rb [hunk 2]
```diff
@@ -0,0 +1,9 @@
+class SlaPolicyDrop < BaseDrop
```
Comment (muhsin-k) [inline app/drops/sla_policy_drop.rb:1]: Should we add the specs? 
cc @sojan-official

File: app/views/mailers/agent_notifications/conversation_notifications_mailer/sla_missed_first_response.liquid (hunk 1 above)
Comment (vishnu-narayanan) [reply app/views/mailers/agent_notifications/conversation_notifications_mailer/sla_missed_first_response.liquid:3]: done

File: app/mailers/agent_notifications/conversation_notifications_mailer.rb [hunk 3]
```diff
@@ -54,14 +54,46 @@ def participating_conversation_new_message(conversation, agent, message)
     send_mail_with_liquid(to: @agent.email, subject: subject) and return
   end
 
+  def sla_missed_first_response(conversation, agent, sla_policy)
```
[the remark this answers is not in the document]
Comment (vishnu-narayanan) [reply app/mailers/agent_notifications/conversation_notifications_mailer.rb:57]: done

File: app/drops/sla_policy_drop.rb (hunk 2 above)
Comment (vishnu-narayanan) [reply app/drops/sla_policy_drop.rb:1]: done

**Comment by muhsin-k** [comment]
@vishnu-narayanan The code looks good to me. Please fix the CE specs.

**Pull request merged**